<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CSV_JSON_Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load_btn">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="csv_input_screen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>531</width>
      <height>655</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="input_textbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Input : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="output_textbox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Output : </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="right_arrow_textbox">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>0</y>
      <width>30</width>
      <height>720</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>‚ûù</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="input_name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>470</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_name">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>470</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="csv_to_json_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>650</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE [CSV  -&gt; JSON]</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="csv_output_screen">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>45</y>
      <width>531</width>
      <height>655</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="json_to_csv_btn">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>650</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>SAVE [JSON  -&gt; CSV]</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="json_output_screen">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>45</y>
      <width>531</width>
      <height>655</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="json_input_screen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>531</width>
      <height>655</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
